src/fillTank.test.js

```diff
@@ -2,10 +1,153 @@
+"use strict";
 
-describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+describe("fillTank", () => {
+  const { fillTank } = require("./fillTank");
 
-  it('should ', () => {
+  it(`should be declared`, () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it(`should be full tank if not 'amount'`, () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 8,
+      },
+    };
+
+    fillTank(customer, 10);
+
+    expect(customer).toEqual({
+      money: 2680,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 40,
+      },
+    });
+  });
+
+  it(`should be full tank if customer want too much 'amount'`, () => {
+    const customer = {
+      money: 2000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 30,
+      },
+    };
+
+    fillTank(customer, 30, 20);
+
+    expect(customer).toEqual({
+      money: 1700,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 40,
+      },
+    });
+  });
+
+  it(`should be fix count 'amount'
+    if 'customer' don't have enough
+    money and without 'amount'`, () => {
+    const customer = {
+      money: 800,
+      vehicle: {
+        maxTankCapacity: 50,
+        fuelRemains: 15,
+      },
+    };
+
+    fillTank(customer, 50);
+
+    expect(customer).toEqual({
+      money: 0,
+      vehicle: {
+        maxTankCapacity: 50,
+        fuelRemains: 31,
+      },
+    });
+  });
 
+  it(`should be fix count 'amount'
+    if 'customer' don't have enough
```

Comment:
Similar to the previous comment, the test description is split across multiple lines. It would be better to combine these lines for clarity.